api/json-vs-xml.md

```diff
@@ -1,31 +1,120 @@
 ---
 description: >-
-  Choose between JSON and XML in the `api-connector` and `order-webhook-connector`
+  Choose between JSON and XML formats in the Tradecloud API. Learn about format support, when to use each, and how to implement both JSON and tXML formats.
 ---
 
 # JSON versus XML
 
-The Tradecloud API by default supports a proprietary JSON format, but for some API endpoints a proprietary XML format is supported.
+The Tradecloud API supports both JSON and XML formats to accommodate different integration requirements and legacy systems. This guide helps you understand the differences and choose the appropriate format for your implementation.
 
-## JSON
+## Overview
 
-The Tradecloud API by default supports a proprietary Tradecloud [JSON format](requests.md#json-body). This is the default for all endpoints.
+The Tradecloud API provides two data formats:
 
-## XML
+- **JSON**: The default format for all API endpoints
+- **tXML**: A proprietary XML format available for specific endpoints
 
-The Tradecloud API supports a proprietary Tradecloud [XML format](requests.md#xml-body), called `tXML`, which is a 1-on-1 translation of the proprietary JSON format:
+Both formats contain identical data structures and functionality - tXML is a one-to-one mapping of the JSON format.
 
-* [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) and
-* [Send single delivery order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSingleDeliveryOrderByBuyerRoute) endpoints support `tXML`.
+## When to use each format
 
-You can see an XML example by selecting "application/xml" in the "Parameter content type" dropdown, under the "Example Value" in above API specifications:
+### Use JSON when
+
+- Building new modern integrations (recommended)
+- Implementing REST API clients
+- You need smaller payload sizes
+
+### Use tXML when
+
+- Integrating with legacy systems that require XML
+- Working with enterprise middleware that requires XML
+- Your existing infrastructure is XML-based
+
+## JSON Format
+
+JSON is the **default and recommended format** for all Tradecloud API endpoints.
+
+### Advantages
```

Comment:
I fined this "Advantages" section a bit of AI-bloat. The bullets mentioned are opinionated, and the customer already got the needed info to make a decision in the paragraph above. imo this section can be removed.

Reply:
I see it's also not consistent:
- no Disadvantages mentioned
- No Advantages / Disadvantages mentioned for tXML

Reply:
removed